Makefile

```diff
@@ -44,6 +45,7 @@ test-memory-growth: build-all-scenarios
 		--dist=loadscope \
 		-n `nproc --ignore 1` \
 		./smarts/core/tests/test_smarts_memory_growth.py
+	rm -f .coverage*
```

Comment:
I meant to add the dot here too.

Reply:
Is that dot needed? I know there is also a file called `.coverage` that is left behind.

Reply:
In general, I want to be careful with `rm -f` commands using wildcards to make sure they don't remove more than intended... the more restrictive the match pattern, the better.  So I think it's a bit better to have the dot just to restrict the matching cases a slight bit more.  (I'm imagining the case where someone did something like `cp .coverage.whatever.1234.5678 .coverage` in order to make a backup of a particularly-interesting coverage file.)

I personally haven't had a bare  `.coverage` file auto-created on my own laptop, but if they are then I guess we could probably do without the final dot.  Or, alternatively, we could just add an additional, explicit `rm -f .coverage` line without a wildcard to handle that case separately.  I guess it's not that big of a deal.

Reply:
thanks for the suggestion.